wagtailmedia/static/wagtailmedia/js/media-chooser-telepath.js

```diff
@@ -1,10 +1,11 @@
-class MediaChooser {
-    constructor(html, idPattern) {
+window.telepath.register('wagtailmedia.MediaChooser', MediaChooser);
```

Comment:
this looks like a rogue register call as the same happesn on line 19

Reply:
Yes, copy & paste error, thx :)